### PR TITLE
Added method for handling DataTable related operations

### PR DESCRIPTION
**Pull request by rohanoid5**

### CODE REVIEW

File: src/FusionCharts.js [hunk 1]
```diff
@@ -228,16 +229,35 @@ export default class ReactNativeFusionCharts extends Component {
     return String(currValue) === String(oldValue);
   }
 
+  dataTableOperations(funcName, args) {
```
Comment (ashok1994) [inline src/FusionCharts.js]: args is ...args, spread it

File: src/FusionCharts.js [hunk 2]
```diff
@@ -228,16 +229,35 @@ export default class ReactNativeFusionCharts extends Component {
...
+    const script = `
+      if(window.dataTable) {
+        var dataTable = window.dataTable;
+        var res;
+        if(${args}) {
+          res = dataTable.${funcName}(${args})
```
Comment (ashok1994) [inline src/FusionCharts.js]: use apply to call that function with datatable as this parameter

File: src/FusionCharts.js (hunk 1 above)
Comment (rohanoid5) [reply src/FusionCharts.js]: @ashok1994 Updated.

File: src/FusionCharts.js (hunk 2 above)
Comment (rohanoid5) [reply src/FusionCharts.js]: @ashok1994 updated.

**Pull request merged**